docs/Makefile

```diff
@@ -11,7 +11,17 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) sou
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
 
-.PHONY: help clean html publish
+# get the most recent tagged version
+# (note that the regex here must be consistent with `smv_tag_whitelist` in the sphinx config)
+OS := $(shell uname)
+ifeq ($(OS), Darwin)
+    LATEST_VERSION = $(shell git for-each-ref --sort=taggerdate --format '%(tag)' | grep -E '^v\d+\.\d+\.\d+$$' | tail -1)
+endif
+ifeq ($(OS), Linux)
+    LATEST_VERSION = $(shell git for-each-ref --sort=taggerdate --format '%(tag)' | grep -P '^v\d+\.\d+\.\d+$$' | tail -1)
+endif
```

Comment:
These two conditionals, and a echo statement for `LATEST_VERSION` on L39 are the only changes I made @keithchev